keras_cv/layers/preprocessing/random_color_jitter.py

```diff
@@ -66,51 +61,59 @@ class RandomColorJitter(VectorizedBaseImageAugmentationLayer):
             is used, a value between `0.0` and the passed float is sampled.
             In order to ensure the value is always the same, please pass a tuple
             with two identical floats: `(0.5, 0.5)`.
+        value_range: Optional list/tuple of 2 floats for the lower and upper limit
+            of the values of the input data. Defaults to [0.0, 255.0]. Can be
+            changed to e.g. [0.0, 1.0] if the image input has been scaled before
+            this layer.
         seed: Integer. Used to create a random seed.
 
     Usage:
     ```python
     (images, labels), _ = tf.keras.datasets.cifar10.load_data()
     color_jitter = keras_cv.layers.RandomColorJitter(
-            value_range=(0, 255),
             brightness_factor=(-0.2, 0.5),
             contrast_factor=(0.5, 0.9),
             saturation_factor=(0.5, 0.9),
             hue_factor=(0.5, 0.9),
+            value_range=(0, 255),
     )
     augmented_images = color_jitter(images)
     ```
     """
 
     def __init__(
         self,
-        value_range,
         brightness_factor,
         contrast_factor,
         saturation_factor,
         hue_factor,
+        value_range=(0, 255),
```

Comment:
remove default

Reply:
A following question:
Should I also remove default value (0, 255) in RandomBrightness, RandomContrast?

Reply:
Done.

And I revert the change of `value_range` order. It is the same as previous implementation.